### PR TITLE
Describe workaround for the autocreate issue, #230

### PR DESCRIPTION
**Pull request by patriknw**
Refs #230

### CODE REVIEW

File: core/src/main/resources/reference.conf [hunk 1]
```diff
@@ -325,9 +333,17 @@ cassandra-snapshot-store {
...
+  # This is convenient in development and test environment, but for production
+  # it's recommended to disable this and create the keyspace and tables manually
+  # (with sa cript) once. Not all Cassandra settings are configurable when using
+  # autocreate.
   keyspace-autocreate = true
 
   # Parameter indicating whether the snapshot tables should be auto created
+  # This is convenient in development and test environment, but for production
+  # it's recommended to disable this and create the keyspace and tables manually
+  # (with sa cript) once. Not all Cassandra settings are configurable when using
```
Comment (raboof) [inline core/src/main/resources/reference.conf]: that 's' travelled! :)

File: core/src/main/resources/reference.conf [hunk 2]
```diff
@@ -325,9 +333,17 @@ cassandra-snapshot-store {
   keyspace = "akka_snapshot"
 
   # Parameter indicating whether the snapshot keyspace should be auto created
+  # This is convenient in development and test environment, but for production
+  # it's recommended to disable this and create the keyspace and tables manually
+  # (with sa cript) once. Not all Cassandra settings are configurable when using
+  # autocreate.
```
Comment (raboof) [inline core/src/main/resources/reference.conf]: This seems a bit hand-wavey to me - "Not all Cassandra settings are configurable when using autocreate" is a bit vague (which settings do we suggest changing?), and if that matters I'd generally recommend doing the same for development/test.

Why would we recommend against autocreating for production? It seems quite convenient and hard to mess up.

File: README.md [hunk 3]
```diff
@@ -97,7 +97,14 @@ The new columns `meta_ser_id`, `meta_ser_manifest`, and `meta` are defined in th
...
+It is also not required to add the materialized views, not even if the meta data is stored in the journal table. If the materialized view is not changed the plain events are retrieved with the `eventsByTag` query and they are not wrapped in `EventWithMetaData`. Note that Cassandra [does not support](http://docs.datastax.com/en/cql/3.3/cql/cql_reference/cqlAlterMaterializedView.html) adding columns to an existing materialized view.
+
+
+If you see exception "Undefined column name meta_ser_id" it is because Cassandra validates the ["CREATE MATERIALIZED VIEW IF NOT EXISTS"](https://docs.datastax.com/en/cql/3.3/cql/cql_reference/cqlCreateMaterializedView.html#cqlCreateMaterializedView__if-not-exists) even though the view already exists and will not be created. To work around that issue you can disable the `tables-autocreate`:
```
Comment (raboof) [inline README.md]: :+1:  this is of course very useful to document

File: core/src/main/resources/reference.conf (hunk 2 above)
Comment (patriknw) [reply core/src/main/resources/reference.conf]: ok, I have reworded. There are many many settings that you may tune for a C* production environment and we have got requests to add them to the autocreate statements, but that will not be possible to maintain.

C* cluster is also not very good at dealing with concurrent creation (from different nodes) of keyspace and tables.

**Pull request merged**